### PR TITLE
Update to containerd 1.3.2

### PR DESCRIPTION
**Pull request by dims**
v1.3.2 fixes some nasty issues including a race condition, please see:

https://github.com/containerd/containerd/releases/tag/v1.3.2

### CODE REVIEW

File: images/capi/packer/config/containerd.json [hunk 1]
```diff
@@ -1,4 +1,4 @@
 {
-    "containerd_version": "1.3.0",
+    "containerd_version": "1.3.2",
     "containerd_sha256": "47653ab55b58668ce93704e47b727b41f57d296e4048d6860daf55d6b7c2bf18"
```
Comment (ncdc) [inline images/capi/packer/config/containerd.json]: Shouldn't the sha change?

Comment (dims) [reply images/capi/packer/config/containerd.json]: whoops! done.

**Comment by ncdc** [comment]
LGTM. I'd squash. I'm guessing we should build some new images with the new containerd version and test them out?

**Comment by dims** [comment]
Done. sorry was working on something and did not want to forget this so had just created this from github UI.

**Pull request merged**